pyanaconda/ui/gui/helpers.py

```diff
@@ -81,6 +115,17 @@ class GUISpokeInputCheckHandler(GUIInputCheckHandler):
 
     __metaclass__ = ABCMeta
 
+    def __init__(self):
```

Comment:
Indeed! Taking a second look on the class I now see it's also used in Source spoke for general input checking. So I've created the `GUISpokePasswordCheckHandler` class, which inherits from `GUISpokeInputCheckHandler` and has all the password checking specific functionality. The password and user spokes now use the new class while the source spoke still uses the old one. :)